packages/ui-utils/src/types.ts

```diff
@@ -126,6 +138,8 @@ export interface Message {
 
   content: string;
 
+  parts: ContentPart[];
```

Comment:
can parts be optional? we could only set them when files are available. that way we don't change what is being sent from client to server for existing setups

Reply:
agreed, done! 

instead of `parts`, it's now `files`, and it's set to optional